Docs: sync breaking changes in migration and rel notes (#55527)
elastic/elasticsearch: docs/reference/migration/migrate_7_7.asciidoc

`DEBUG` level by default. This sometimes resulted in a good deal of unnecessary
log noise. From 7.7 onwards the default log level for logger in this hierarchy
is now `INFO`, in line with most other loggers. If needed, you can recover the
pre-7.7 default behaviour by adjusting your {ref}/logging.html[logging].

[discrete]
[[breaking_77_mapping_changes]]
=== Mapping changes

[discrete]
[[stricter-mapping-validation]]
==== Validation for dynamic templates

> So far misconfiguration of dynamic templates have been discovered when indexing
> a document with an unmapped field only. In 8.0 and later the dynamic mappings
> have stricter validation done at mapping update time and invalid updates - e.g.
> incorrect analyzer settings or unknown field types used - are failed. For
> indices created in 7.7 and later the update succeeds, but a warning is emitted.

Reviewer: ```suggestion
So far misconfiguration of dynamic templates have been discovered when indexing
a document with an unmapped field only. In {es} 8.0 and later versions, dynamic mappings
have stricter validation, done at mapping update time. Invalid updates, such as using
incorrect analyzer settings or unknown field types, fail. For
indices created in {es} 7.7 and later version, the update succeeds but emits a warning.
```

is now `INFO`, in line with most other loggers. If needed, you can recover the

> pre-7.7 default behaviour by adjusting your <<logging>>.

Reviewer: Because this is now being reused in the Installation and Update Guide, any internal xrefs
need to be made into external links. This should fix the docs CI test.
```suggestion
pre-7.7 default behaviour by adjusting your {ref}/logging.html[logging].
```

Author: Thank you!!
I did spot the warning, but since it built (and linked) fine locally, I dismissed it as potential failure source (and was just trying to build `--all` to hopefully get more details).

do this. The behavior is now the same for both field types like documented in

> <<range-query-date-math-rounding>>.

Reviewer: ```suggestion
{ref}/query-dsl-range-query.html#range-query-date-math-rounding[Date math and rounding].
```
